src/core/Akka.Remote.Tests/Transport/HeliosTransportDnsResolutionSpec.cs

```diff
@@ -128,14 +129,22 @@ public static bool IsAnyIp(EndPoint ep)
         }
 
         [Property()]
-        public Property HeliosTransport_Should_Resolve_DNS(EndPoint inbound, EndPoint outbound, bool dnsIpv6, bool enforceIpFamily, bool monoRuntime)
+        public Property HeliosTransport_Should_Resolve_DNS(EndPoint inbound, EndPoint outbound, bool dnsIpv6, bool enforceIpFamily)
         {
+            // TODO: Mono does not support IPV6 Uris correctly https://bugzilla.xamarin.com/show_bug.cgi?id=43649 (Aaronontheweb 8/22/2016)
+            if (IsMono)
```

Comment:
Anywhere we're using IPV6, whose addresses can't be parsed correctly by the `Uri` class on Mono 4.4.2, we have to filter out those test cases. Mono 4.6 will fix this.